test/test_serde.py

```diff
@@ -10,6 +10,7 @@
 from torch import Tensor
 
 import syft
+from syft.frameworks.torch.tensors.interpreters.additive_shared import AdditiveSharingTensor
```

Comment:
Why is this one needed? just calling syft. AdditiveSharingTensor doesn't work?

Reply:
or, are you preparing the future? :)

Reply:
yes, this change was from when I was hoping to remove tensors from the main `syft` namespace. I can change it back if we want!

Reply:
Let's keep it for now :)